source/interaction/ifed_method.cc

```diff
@@ -652,36 +652,16 @@ namespace fdl
         // available
         position.update_ghost_values();
         velocity.update_ghost_values();
-        for (auto &force : part.get_stress_contributions())
-          force->setup_force(data_time, position, velocity);
-        for (auto &force : part.get_volumetric_force_contributions())
-          force->setup_force(data_time, position, velocity);
-        for (auto &force : part.get_boundary_force_contributions())
+        for (auto &force : part.get_force_contributions())
```

Comment:
In particular, the implicit mechanics code should more-or-less duplicate this part of `fdl::IFEDMethod`.